src/api.v2/helpers/pipeline/shouldPipelineRerun.js

```diff
@@ -89,16 +91,33 @@ const getGem2sParams = async (experimentId, rawSamples = undefined) => {
   };
 };
 
-const shouldGem2sRerun = async (experimentId, pipelineType) => {
+const getQCParams = async (experimentId) => {
+  const metadataResult = (
+    await new CellLevelMeta().getMetadataByExperimentIds([experimentId])
+  )[0] || {};
+
+  const cellMetadataId = metadataResult.id || null;
+
+  return { cellMetadataId };
+};
```

Comment:
this function contains basically the same functionality as the one I created and merged as part of this ticket: https://github.com/hms-dbmi-cellenics/issues/issues/38 I think you should pull master branch of API